apps/webapp/app/services/email.server.ts

```diff
@@ -1,5 +1,6 @@
 import type { DeliverEmail, SendPlainTextOptions } from "emails";
 import { EmailClient } from "emails";
+import { MailTransportOptions } from "emails/transports";
```

Comment:
This causes a typecheck error because it's trying to access something not exported from the main file and it's not specified as a separate export in the package.json file.

I'd just re-export `MailTransportOptions` from the index file and change this import to be from `emails`.

Reply:
Gah, there it is... thank you!